modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm

```diff
@@ -20,8 +20,9 @@ GLOBAL_LIST_EMPTY(ash_rituals)
 	if(!current_ritual)
 		. += span_notice("<br>There is no selected ritual at this moment-- use the central rune to select a ritual.")
 		return
-	. += span_notice("<br>The current ritual is: [current_ritual.name]<br>")
-	. += span_warning("The required components are as follows:")
+	. += span_notice("<br>The current ritual is: [current_ritual.name]")
```

Comment:
`</br>`?

Reply:
Break line, it creates a space between the things.